docs/functions-develop-security.md

```diff
@@ -2,19 +2,22 @@
 id: functions-develop-security
 title: Enable security on functions
 sidebar_label: "Enable security on functions"
+description: Learn to enable security on functions in Pulsar.
 ---
 
 ````mdx-code-block
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ````
 
+If you want to enable security on functions, complete the following steps:
+
 ## Prerequisites
 
-If you want to enable security on functions, you need to [enable security settings](functions-worker.md) on function workers first.
+- you need to [enable security settings](functions-worker.md) on function workers first.
```

Comment:
I believe the original text is more accurate because "enable security" might be optional.